### PR TITLE
libmatheval: init at 1.1.11

### PR DESCRIPTION
**Pull request by bzizou**
###### Things done:
- [ ] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [X] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
###### More

Fixes issue #<insert id>

cc @<maintainer>

---

_Please note, that points are not mandatory, but rather desired._

### CODE REVIEW

File: pkgs/development/libraries/libmatheval/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, guile, autoconf, flex }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.11";
+  name = "libmatheval-${version}";
+
+  buildInputs = [ guile autoconf flex ];
```
Comment (joachifm) [inline pkgs/development/libraries/libmatheval/default.nix]: I think at least `autoconf` and `flex` could be moved to `nativeBuildInputs`, if they are not used at run-time somehow.

**Comment by joachifm** [comment]
This looks good to me. My only concern is that you're checking in two rather large diffs into nixpkgs; is it possible to use `fetchpatch` or similar instead?

**Comment by bzizou** [comment]
Thanx for telling me the correct method! I amended my commit!

**Comment by joachifm** [comment]
Works for me. Thank you

**Pull request merged**